README.md

```diff
@@ -16,58 +13,17 @@ maven
    cd SER516-Team-Kansas
    ```
 
-### 2) Compile and Run the spring boot application
-Go to the project root and compile the Maven project
-to install dependencies run:
-```bash
-mvn clean install -U
```

Comment:
could we not delete this?, could we keep this as documentation?
I see the value of having the simple docker compose command but its still better if we have this at the bottom and the docker command at the top.

Reply:
The project relies on docker to run properly at this point. Keeping these commands is misleading and useless.